src/main/java/io/anserini/index/IndexReaderUtils.java

```diff
@@ -892,13 +893,16 @@ public static void main(String[] argv) throws Exception {
     IndexReader reader = IndexReaderUtils.getReader(args.index);
     Map<String, Object> results = IndexReaderUtils.getIndexStats(reader, args.field);
 
+    Path indexPath = getIndex(args.index);
     if (args.stats) {
       System.out.println("Index statistics");
       System.out.println("----------------");
       System.out.println("documents:             " + results.get("documents"));
       System.out.println("documents (non-empty): " + results.get("non_empty_documents"));
       System.out.println("unique terms:          " + results.get("unique_terms"));
       System.out.println("total terms:           " + results.get("total_terms"));
+      System.out.println("physical location:     " + indexPath.toAbsolutePath());
+      System.out.println("total size on disk:    " + new File(indexPath.toString()).length() + " bytes");
```

Comment:
you probably want to gather this in `Map<String, Object> results = IndexReaderUtils.getIndexStats(reader, args.field);`? So that other calls to `getIndexStats` will also have these kv pairs?